common/compiler/solidity_test.go

```diff
@@ -26,7 +26,7 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-const solcVersion = "0.9.23"
+const solcVersion = "0.1.1"
```

Comment:
`0.9.23` was the version of cpp-ethereum. At the point solc was actually released, it started with `0.1.0`.